lib/models/information/information_template.dart

```diff
@@ -60,36 +93,32 @@ class CPInformationTemplate extends CPTemplate {
         'backButton': backButton?.toJson(),
         'actions': actions.map((e) => e.toJson()).toList(),
         'informationItems': informationItems.map((e) => e.toJson()).toList(),
-        'leadingNavigationBarButtons':
-            leadingNavigationBarButtons.map((e) => e.toJson()).toList(),
-        'trailingNavigationBarButtons':
-            trailingNavigationBarButtons.map((e) => e.toJson()).toList(),
+        'leadingNavigationBarButtons': leadingNavigationBarButtons.map((e) => e.toJson()).toList(),
+        'trailingNavigationBarButtons': trailingNavigationBarButtons.map((e) => e.toJson()).toList(),
       };
 
   /// Update the properties of the [CPInformationTemplate]
-  void update({
+  Future<CPInformationTemplate> update({
```

Comment:
please state what happens if the `update` was not successful

Reply:
done